doc/release-notes.md

```diff
@@ -84,15 +105,80 @@ implied blockmaxweight, instead of limiting block size directly. Any miners who
 to limit their blocks by size, instead of by weight, will have to do so manually by
 removing transactions from their block template directly.
 
+
+GUI settings backed up on reset
+-------------------------------
+
+The GUI settings will now be written to `guisettings.ini.bak` in the data directory before wiping them when
+the `-resetguisettings` argument is used. This can be used to retroactively troubleshoot issues due to the
+GUI settings.
+
+
+Duplicate wallets disallowed
+----------------------------
+
+Previously, it was possible to open the same wallet twice by manually copying the wallet file, causing
+issues when both were opened simultaneously. It is no longer possible to open copies of the same wallet.
+
+
+Debug `-minimumchainwork` argument added
+----------------------------------------
+
+A hidden debug argument `-minimumchainwork` has been added to allow a custom minimum work value to be used
+when validating a chain.
+
+
 Low-level RPC changes
 ----------------------
+
 - The "currentblocksize" value in getmininginfo has been removed.
 
+- `dumpwallet` no longer allows overwriting files. This is a security measure
+  as well as prevents dangerous user mistakes.
+
+- `backupwallet` will now fail when attempting to backup to source file, rather than
+  destroying the wallet.
+
+- `listsinceblock` will now throw an error if an unknown `blockhash` argument
```

Comment:
May want to clarify that its still allowed to be an empty string, which will still have the old behavior.